npm/util/util.go

```diff
@@ -238,3 +252,42 @@ func DropEmptyFields(s []string) []string {
 
 	return s
 }
+
+// GetNSNameWithPrefix returns Namespace name with ipset prefix
+func GetNSNameWithPrefix(nsName string) string {
+	return NamespacePrefix + nsName
+}
+
+// CompareResourceVersions take in two resource versions and returns true if new is greater than old
+func CompareResourceVersions(rvOld string, rvNew string) bool {
+	// Ignore oldRV error as we care about new RV
+	tempRvOld := ParseResourceVersion(rvOld)
+	tempRvnew := ParseResourceVersion(rvNew)
+	if tempRvnew > tempRvOld {
+		return true
+	}
+
+	return false
+}
+
+// CompareUintResourceVersions take in two resource versions as uint and returns true if new is greater than old
+func CompareUintResourceVersions(rvOld uint64, rvNew uint64) bool {
+	if rvNew > rvOld {
+		return true
+	}
+
+	return false
+}
+
+// ParseResourceVersion get uint64 version of ResourceVersion
+func ParseResourceVersion(rv string) uint64 {
+	if rv == "" {
+		return 0
+	}
+	rvInt, err := strconv.ParseUint(rv, 10, 64)
```

Comment:
should potentially store resource version as uint if we're comparing every time, can take that in a later change